docs/source/graphicaldesktop/graphicaldesktop.rst

```diff
@@ -57,6 +57,22 @@ Please note that users are only permitted to connect to your Flight Compute clus
 Resizing the desktop to fit your screen
 ---------------------------------------
 
+Specifying a size with the alces session tool
+*********************************************
+
+When launching a graphical desktop session using the ``alces session`` utility - a session resolution can be specified using the ``--geometry <size>`` option. A similar effect can be achieved by launching a session with no ``--geometry`` option set will launch a session with a default size of ``1024x768``, then manually changing the size of the session using the ``xrandr`` command from your graphical session. 
```

Comment:
I'm not entirely sure why but these aren't completely equivalent - for instance if you start a session with a small geometry and then run `xrandr` to increase this within a session, the new resolution doesn't seem to be picked up by all programs and so some text can appear oversized. Whereas things appear sized correctly if you specify the geometry when starting the session. Not sure if this is something we want to mention somehow in the docs though.